src/bci_build/package/basecontainers.py

```diff
@@ -29,6 +29,12 @@ def _get_os_container_package_names(os_version: OsVersion) -> tuple[str, ...]:
     return ("sles-release",)
 
 
+def _get_eula_package_names(os_version: OsVersion) -> str:
```

Comment:
The type hint is wrong
```suggestion
def _get_eula_package_names(os_version: OsVersion) -> Tuple[str...]:
```